conans/client/command.py

```diff
@@ -735,7 +735,8 @@ def run(self, *args):
             self._user_io.out.error(msg)
         except Exception as exc:
             import traceback
-            print(traceback.format_exc())
+            # print(traceback.format_exc())
```

Comment:
I think we kept it commented to avoid the problems with the Cyrillic characters...